proposals/06-poc-projects-with-tutorials.md

```diff
@@ -0,0 +1,110 @@
+
+#### Assumptions &amp; hypotheses
+
+- Mature enough tech that can be used to build POC apps
+  - This probably means that libp2p and IPFS are more preferable targets to start with for now as opposed to Filecoin
+- Tutorials in form of articles are a very good entry point for developers to read on new ways to build apps
+
+#### User workflow example
+
+Developers would most likely start with the tutorial/article that details how we can create X app using libp2p and/or IPFS.
+The app should be deployed and publicly accessible for devs to play with: this is the proof that our stack works so it needs to be fairly bug-free and stable.
+
+#### Impact
+
+🔥🔥🔥
+
+Developers would be able to gain clear confidence in our stack because they saw with their own eyes that it works. If we invest on making a fair share of POCs, we can always respond to questions of the form "Can we do this with IPFS?" with an url to the POC app.
+
+More importantly, if the apps really succeed, they might even spin off into their own projects and succeed and solving the issue they tackle in such an amazing way that everyone wants to use it and contribute to improve it.
+
+#### Leverage
+
+🎯🎯🎯
+
+POCs for specific use cases can bump into limitations or even bugs in the stack. This would contribute to our objective of improving the stack by providing this feedback to the stack development in order to prioritize issues to be solved.
+We wouldn't publish apps that do not work, but we would be bumping up the importance of specific issues that arise from building such apps.
+
+#### Confidence
+
+**Impact=8**
+
+**Confidence=7**
+
+**Ease=6**
+
+**ICE=7**
+
+The end result would most likely very well received by developers. The major limitations would be driven by what our stack currently can achieve.
+
+## Project definition
+#### Brief plan of attack
+
+Brainstorm some ideas for POCs. Build one, deploy and publish the tutorial and analyse impact.
+Create guides and other helpers to incentivize POC building and publishing.
+Automate anonymous data collection to analyze impact.
+
+#### What does done look like?
+
+1. First POC Project and Tutorial built and published to show the impact
+1. Detailed guide for building more POC Projects and Tutorials
+    - Would focus on making sure the projects are a good fit for this purpose and guide them on specific nuances of building these apps and tutorials.
+1. UI kit guide or even a design system to reduce visual UI implementation to almost zero.
+1. Process for people to submit ideas, build and publish.
+1. Have a plan for releasing the POCs and tutorials on a frequent schedule.
+1. Streamlined process for analyzing impact with specific metrics
+    - Since we will own the apps, we can easily track the user journey from the tutorial to the app and request feedback from the user.
+
+
+####  What does success look like?
+
+POC apps get good traffic and response from the tutorials are mostly positive.
```

Comment:
The ultimate goal we’re shooting for here is successful developers getting great value from the stack. Therefore I’d frame success in “time to build a capable POC”, developer pass through vs bounce, and number of devs flowing through this tutorial (sessions, successful completions, etc).

Reply:
I’d make it a non-goal to create such good POCs that they get organic usage/communities and focus instead of them being a great onboarding tool!

Reply:
Makes sense! What do you think about this:

```suggestion
- Time to build a capable POC is shortened
- Positive developer adoption by reducing bounces
- Good completion rate for completing the tutorial

```